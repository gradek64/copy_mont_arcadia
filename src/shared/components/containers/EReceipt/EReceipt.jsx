import PropTypes from 'prop-types'
import React, { Component } from 'react'

import SandBox from '../SandBox/SandBox'
import AccountHeader from '../../common/AccountHeader/AccountHeader'
import cmsConsts from '../../../constants/cmsConsts'

export default class EReceipts extends Component {
  static contextTypes = {
    l: PropTypes.func,
  }
  render() {
    const { l } = this.context
    return (
      <section>
        <AccountHeader
          link="/my-account"
          label={l`Back to My Account`}
          title={l`E-receipts`}
          classNames=" screen-reader-text"
        />
        <SandBox
          contentType={cmsConsts.PAGE_CONTENT_TYPE}
          cmsPageName="eReceipts"
          {...this.props}
        />
      </section>
    )
  }
}
