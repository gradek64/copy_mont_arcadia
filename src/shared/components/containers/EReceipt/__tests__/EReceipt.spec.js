import testComponentHelper from 'test/unit/helpers/test-component'
import EReceipt from '../EReceipt'

describe('<EReceipt /> ', () => {
  const initialProps = {
    fakeProp: 'fake prop',
  }
  const renderComponent = testComponentHelper(EReceipt)
  describe('@renders', () => {
    it('in default state', () => {
      expect(renderComponent(initialProps).getTree()).toMatchSnapshot()
    })
  })
})
