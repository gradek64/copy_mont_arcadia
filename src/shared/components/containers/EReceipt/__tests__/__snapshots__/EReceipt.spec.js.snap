// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EReceipt />  @renders in default state 1`] = `
<section>
  <AccountHeader
    classNames=" screen-reader-text"
    label="Back to My Account"
    link="/my-account"
    title="E-receipts"
  />
  <Connect(SandBox)
    cmsPageName="eReceipts"
    contentType="page"
    fakeProp="fake prop"
  />
</section>
`;
